src/main/java/edu/kis/powp/jobs2d/drivers/Job2dDriverDecorator.java

```diff
@@ -8,18 +8,18 @@
 import static java.lang.Math.pow;
 import static java.lang.Math.sqrt;
 
-public class UsageMonitor implements Job2dDriver {
+public class Job2dDriverDecorator implements Job2dDriver {
```

Comment:
Now the name does not reflect the UsageMonitor responsibility. Try something in between.